### PR TITLE
Individual result mods

### PR DESCRIPTION
**Pull request by mayerm94**
Saving was failing because keys in statment_results and clause results hashs had periods in them
Pull requests into cqm-models require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/CYPRESS-514
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [ ] If applicable, the library version number in `package.json` and `cqm-models.gemspec` has been updated
- [x] Cqm-execution fixtures have been updated with the update_cqm_execution_fixtures.sh script inside server-scripts using this branch in the cqm-converter https://github.com/projecttacoma/cqm-execution/pull/48

**Bonnie Reviewer:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Cypress Reviewer:**

Name: @dczulada
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: app/assets/javascripts/cqm/ClauseResult.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+const [String, Mixed] = [
+  mongoose.Schema.Types.String,
+  mongoose.Schema.Types.Mixed,
+];
+
+const ClauseResultSchema = mongoose.Schema({
+  // Library the clause this result is for is in
+  library_name: String,
+  // Statment the clause this result is for is in
```
Comment (losborne) [inline app/assets/javascripts/cqm/ClauseResult.js]:
```suggestion
  // Statement the clause this result is for is in
```

File: app/assets/javascripts/cqm/StatementResult.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+const [String, Mixed] = [
+  mongoose.Schema.Types.String,
+  mongoose.Schema.Types.Mixed,
+];
+
+const StatementResultSchema = mongoose.Schema({
+  // Library the statement this result is for is in
+  library_name: String,
+  // Statment this result is for is in
```
Comment (losborne) [inline app/assets/javascripts/cqm/StatementResult.js]:
```suggestion
  // Statement this result is for is in
```

**Comment by hossenlopp** [comment]
Please update and test the cqm-execution and bonnie branches with the latest rebase/commit of this branch so we can test them too.

**Comment by losborne** [comment]
Needs update after https://github.com/projecttacoma/cqm-models/pull/118 getting in (bubbled up to cqm-execution and bonnie)

**Pull request merged**